ee/modules/500-operator-trivy/openapi/config-values.yaml

```diff
@@ -1,5 +1,13 @@
 type: object
 properties:
+  enableReportUpdater:
```

Comment:
Converting to what? Let's describe not the action but the result.

Example:
Action -> Enable report updater
Result -> Add links to BDU to all reports

Reply:
Renamed to `linkCVEtoBDU`